setup.py

```diff
@@ -135,6 +135,7 @@ def run(self):
             # Required to serve models through MLServer
             "mlserver>=0.5.3",
             "mlserver-mlflow>=0.5.3",
+            "virtualenv",
```

Comment:
Shall we restrict the `virtualenv` version ? e.g. `virtualenv>=xx` ?

Reply:
@WeichenXu123 why?

Reply:
I am not sure whether there're some knowns bugs or issues in lower `virtualenv` version, have we fully test on older versions ?

Reply:
No, but I don't think we need to pin virtualenv.